### PR TITLE
Added ossec.server docker image

### PR DESCRIPTION
**Pull request by mixua**
#### Moving the OSSES server to k8s cluster.

Task according to [ch68773]

https://app.clubhouse.io/vgs/story/68773/ossec-create-a-image-and-test-with-the-latest-copy-of-verygood-ossec-server

### CODE REVIEW

**Comment by akhon** [comment]
initially - looks great.
i would remove monit, suricata etc services and rules since we won't need them in k8s.

also, migration to latest ossec-server and to ubuntu18.04 as a base would be good to have now

**Comment by mixua** [comment]
> initially - looks great.
> i would remove monit, suricata etc services and rules since we won't need them in k8s.
> 
> also, migration to latest ossec-server and to ubuntu18.04 as a base would be good to have now

Done

File: image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+FROM williamyeh/ansible:ubuntu18.04
```
Comment (akhon) [inline image/Dockerfile:1]: btw, we have 20.04, shouldn't be a problem right now to migrate there, but would be good to have everything aligned

File: image/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+  wazuh-server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: quay.io/verygoodsecurity/ossec-server:${IMG_TAG}
```
Comment (akhon) [inline image/docker-compose.yaml:8]: please move to ecr, we're migrating there, so all new images have to be in ECR

**Comment by akhon** [comment]
also, link to the task in main description looks outdated :)

**Pull request merged**